test/basic_test.cc

```diff
@@ -99,4 +99,12 @@ void BM_empty_stop_start(benchmark::State& state) {
 BENCHMARK(BM_empty_stop_start);
 BENCHMARK(BM_empty_stop_start)->ThreadPerCpu();
 
+template <class ...Args>
+void BM_with_args(benchmark::State& state, Args&&...) {
```

Comment:
Nevermind this change is needed. GCC 4.6 is a C++0x compiler and so BENCHMARK_CAPTURE isn't defined for it.